freemocap/gui/qt/actions_and_menus/actions.py

```diff
@@ -9,6 +10,11 @@
 REBOOT_GUI_ACTION_NAME = "Reboot GUI"
 EXIT_ACTION_NAME = "Exit"
 
+DOCUMENTATION_ACTION_NAME = "Open Documentation"
+ABOUT_US_ACTION_NAME = "About Us"
+
+DONATE_ACTION_NAME = "Donate to Freemocap"
```

Comment:
Make sure these action names match their connected part of the Menu (not saying they don't now because I didn't check, just saying to make sure they match :)  )

Reply:
They do now!